hawc/apps/animal/exports.py

```diff
@@ -688,21 +688,18 @@ def _func(group_df: pd.DataFrame) -> pd.Series:
             .reset_index(drop=True)
         )
 
-    def handle_treatment_period(self, df: pd.DataFrame):
-        txt = df["experiment-type_display"].str.lower()
-        txt_index = txt.str.find("(")
-        txt_updated = (
-            txt.to_frame(name="txt")
-            .join(txt_index.to_frame(name="txt_index"))
-            .apply(
-                lambda x: x["txt"] if x["txt_index"] < 0 else x["txt"][: x["txt_index"]],
-                axis="columns",
-                result_type="reduce",
-            )
-        ).astype(str)
-        df["treatment period"] = (
-            txt_updated + " (" + df["dosing_regime-duration_exposure_text"]
-        ).where(df["dosing_regime-duration_exposure_text"].str.len() > 0) + ")"
+    def handle_treatment_period(self, df: pd.DataFrame) -> pd.DataFrame:
+        def _calc(row):
+            txt = row["experiment-type_display"].lower()
+            if txt.find("(") >= 0:  # TODO - remove extra space after confirming changes
```

Comment:
If this TODO was addressed we should remove it.

Reply:
resolved in dcd0d0c8f1857de6c2de9eade1ff00cb6ac9be46